_posts/2019-07-12-barcelona-announcement.md

```diff
@@ -0,0 +1,82 @@
+
+The venue is wheelchair accessible over the lateral street and can be reached easily by public transport. For more information about accesss for people with a disability in Barcelona you can take a look at [this link](https://www.barcelona-tourist-guide.com/en/transport/disabled/access-barcelona-disabled.html).
+
+The best options to get there, if you come from Sants Estació, are the [Av. Pedralbes - Cavallers bus station](https://goo.gl/maps/QeNjywZ749Njur8S8) and the [Av d’Esplugues - Av Pearson bus station](https://www.google.com/url?q=https://www.google.com/maps/dir/Sants%2BEstaci%25C3%25B3,%2BBarcelona/Av%2Bd%27Esplugues%2B-%2BAv%2BPearson,%2B08034%2BBarcelona/@41.386477,2.1192745,15z/data%3D!4m15!4m14!1m5!1m1!1s0x12a4987e1c029355:0xb19606f1fb16429e!2m2!1d2.1409712!2d41.3816151!1m5!1m1!1s0x12a4984e1e97de21:0x992b87b41d670e10!2m2!1d2.111625!2d41.3921394!3e3!5i3!5m1!1e2&sa=D&ust=1562873148055000&usg=AFQjCNE1co3JRi_anes-tAbTSK8FFGFX4g); also there’s a subway option to [Palau Reial station](https://www.google.com/url?q=https://goo.gl/maps/4u1dJbpRUHRrrmxE8&sa=D&ust=1562873148056000&usg=AFQjCNHwY0Vd1hK9hNsxd68fmNxsLdHs1A) if you want a quicker route that will require some walk.
+
+Anyway, don't worry, we will update our website soon with plenty of information on this regard, so stay tuned! ;P
+
+## Catering
+
+We will provide breakfast, coffee, drinks and lunch the first two conference days so you don't have to worry about looking for a restaurant.
+
+We'll also have vegan and vegetarian options and, if you have other dietary needs, please just contact us and we'll be happy to help you.
+
+For the *Impl Days*, it will be up to you to decide where to eat. But don't worry, we'll gather information about some good restaurants in the area and we will fill our website with it so you can experience the best of our great gastronomy.
+
+## Ticket information
+
+Two days of talks and workshops, two more days of open space and hacking, breakfast, coffee, drinks and meals... wait, this surely must be very expensive!
+
+Nope! We have managed to get a reasonable ticket price thanks to the [UPC](https://www.upc.edu/en) venue and our sponsors!
+
+The price for the tickets will be:
+
+- 50€ student ticket,
+- 100€ standard ticket,
+- 200€ supporter ticket.
+
+If you are a student or you belong to an underrepresented community, there will be a scholarship programme where you can apply for free tickets, travel and accomodation, too.
+
+So, basically, money hopefully won't be a reason not to come! ;)
+
+## Free RustBridge workshop
+
+As usual, we'll also be hosting at least one [RustBridge workshop](https://rustbridge.com/) (probably two :P) so if you want to learn Rust, join the community and you belong to underrepresented communities and probably have a background in another programming language this is the perfect workshop for you. That being said, note that this is an open workshop for beginners and everybody is welcome to attend.
```

Comment:
We need to talk to the RustBridge folks. I expect it to be the usual workshop spread across 2 days.

I think simplifying this and citing their self-description is better here:
```suggestion
As usual, we'll also be hosting a [RustBridge workshop](https://rustbridge.com/).
RustBridge is a workshop focused on getting underrepresented people with a background in another programming language to learn Rust and join the community.
```